userspace/libsinsp/filterchecks.cpp

```diff
@@ -4541,7 +4550,10 @@ uint8_t* sinsp_filter_check_tracer::extract(sinsp_evt *evt, OUT uint32_t* len, b
 				}
 			}
 
-			*len = strlen(res);
+			if(res)
```

Comment:
If the m_argid is somehow out of range, res will not get initialized. This is basically the same as some of the code that we crashed on further below. I'm not entirely sure yet why m_argid would be out of range but reading the code it seems acceptable to just return NULL, so the added strlen() should not assume res is set.
In general I'm a big proponent of more defensive programming. In cases like this it would only add very few cycles to sanity check, but may help prevent more core dumps in the future.